sdk/core/Azure.Core.Experimental/src/RequestOptions.cs

```diff
@@ -13,10 +14,14 @@ namespace Azure
     /// </summary>
     public class RequestOptions
     {
+        private List<HttpMessageClassifier> _classifiers = new();
```

Comment:
Can we make this feature free till it's used? i.e. no allocate before somebody adds clasifiers.

Reply:
Sure.